src/types.ts

```diff
@@ -19,9 +19,7 @@ export type AggregateType = {
   ClassMember: swc.ClassMember
   Declaration: swc.Declaration
   DefaultDecl: swc.DefaultDecl
-  ExportSpecifier: swc.ExportSpecifier
```

Comment:
What was the reason for removing these from the `AggregateType`? Same for `swc.ImportSpecifier` below?

Reply:
I was having conflict between `ExportSpecifier` the aggregate type and `NamedExportSpecifier` whose type is `"ExportSpecifier"`
I'm still planning on restoring theme